### ISSUE TITLE
Why there have a coefficient "2" in the loss self.TVLoss_weight*2*(h_tv/count_h+w_tv/count_w)/batch_size? 

### ISSUE THREAD

**Issue by ghost** [jxgu1016/Total_Variation_Loss.pytorch#1]
I am very confused with it. Could you explain about it? Thank you very much!

**Comment by liuneng1994** [comment]
h_tv and w_tv are quadratic sum, their derivative is 1/2,  in the backward, 2 * 1/2 =1 , many loss function  
use this tip.

**Comment by liuneng1994** [comment]
应该是个中国人,导数1/2乘以2能抵消掉这样能减少小数计算.

**Comment by FantasyJXF** [comment]
这个总变差损失的实现和TF版快速风格迁移完全一样啊，神了

**Comment by DOU0112** [comment]
> 应该是个中国人,导数1/2乘以2能抵消掉这样能减少小数计算.

平方求导导数为什么不是2呀
